### PR TITLE
Add Component on Feature-components Branch

### PR DESCRIPTION
**Pull request by Anvi98**
- Delete all the boilerplate from CRA (text, images, styles).
- Create a directory called `components`.
- Inside `components`, create a new `Calculator.js` file.
- In `Calculator.js`, create a React component that matches the design.

- Don't add any extra design.
- Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
- Use class-based components.
- Once the Calculator is ready, import it and use it in your main component (App.js).
- Test it in the browser; it should render without problems.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,11 @@
...
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
```
Comment (DeliceLydia) [inline src/App.js:8]: - [x] The `App.js` component should be class-based please work on that

File: src/index.js [hunk 2]
```diff
@@ -1,17 +1,17 @@
+/* eslint-disable no-unused-vars */
+
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
 import App from './App.js';
+import Calculator from './components/Calculator.js';
 import reportWebVitals from './reportWebVitals.js';
```
Comment (DeliceLydia) [inline src/index.js]: - [x] Please remove all the boilerplate that was created that you will not use in your app like(
texts, images, and styles)

File: src/components/Calculator.css [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  background-color: #fe9241;
+  display: flex;
+  flex: 1;
+  color: #fff;
+}
+
+.input-calc {
+  justify-items: right;
+  width: 100%;
+}
```
Comment (DeliceLydia) [inline src/components/Calculator.css]: - [x] Try to follow the given design

Design
design

yours
yr

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
# Hi @Anvi98👏👏, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._


### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Anvi98** [comment]
Except for the picture, all your reviews are from the dev branch and not the current-branch feature components.
Could you please check the current branch?

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashbaord. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**